tests/test_vault_conditions.py

```diff
@@ -74,13 +74,11 @@ def test_token_approval_for_vault(allowlist_factory, allowlist, owner, origin_na
 #   - zap_in_pickle_address: "0xc695f73c1862e050059367B2E64489E66c525983"
 #   - another address on the custom 
 def test_token_approval_for_zap(allowlist_factory, allowlist, owner, network_variables, origin_name, implementation, vault_token):
-    # This test is not supported on all networks
-    test_supported = "zap_in_to_vault_address" in network_variables
-    if (chain.id == 1):
-        assert test_supported == True
-    if (test_supported == False):
-        return
-    zap_in_contract = Contract(network_variables["zap_in_to_vault_address"])
+    zap_in_to_vault_address_key = "zap_in_to_vault_address"
+    if zap_in_to_vault_address_key not in network_variables:
+        pytest.skip("no zap in address on this network")
```

Comment:
There must be a way to generalize this logic into a fixture, just trying to work it out but not essential